src/styles/blocks/button.scss

```diff
@@ -0,0 +1,25 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  box-sizing: border-box;
+  width: 100%;
+  height: 56px;
+  background: $c-white;
+  border-radius: 40px;
+
```

Comment:
is this black border needed here?

![black-border](https://user-images.githubusercontent.com/89162624/209443027-15827291-113a-4abd-acb3-e415a679a15e.png)

Reply:
fixed